client/components/popover/index.jsx

```diff
@@ -38,6 +38,7 @@ class PopoverInner extends Component {
 		className: '',
 		closeOnEsc: true,
 		isRtl: false,
+		isFocusEnabled: false,
```

Comment:
As the popover is currently always focussed, can we default this to `true`, then pass in `isFocusEnabled={ false }` for the user mentions popover?

Reply:
@bluefuton Have made the change! 👍